pom.xml

```diff
@@ -179,7 +179,7 @@
     <antlr4.version>4.5.3</antlr4.version>
     <jpam.version>1.1</jpam.version>
     <selenium.version>2.52.0</selenium.version>
-    <paranamer.version>2.8</paranamer.version>
+    <paranamer.version>2.6</paranamer.version>
```

Comment:
Although we defined the version to 2.8, but it is not used this variable. So, we has been using 2.3.